constants/create_post.py

```diff
@@ -6,3 +6,6 @@
 
 SUCCESS_MESSAGE_TEXT = "New post successfully created."
 SUCCESS_MESSAGE_XPATH = f"//*[contains(text(), '{SUCCESS_MESSAGE_TEXT}')]"
+
+EMPTY_FIELDS_ERRORS = {"en": "Please fill out this field.",
+                       "ru": "Заполните это поле."}
```

Comment:
This change needs to be removed, since we did not implement automatic language selection => this will not work in the existing code